src/uwtools/tests/drivers/test_driver.py

```diff
@@ -3,9 +3,31 @@
 Tests for uwtools.drivers.driver module.
 """
 
+from importlib import resources
+from unittest.mock import patch
+
+import pytest
+
 from uwtools.drivers import driver
+from uwtools.drivers.driver import Driver
+from uwtools.tests.support import fixture_path
 
 
 def test_import():
     # Test stub asserting that module was imported.
     assert dir(driver)
+
+
+@pytest.mark.parametrize("cls", Driver.__subclasses__())
+@pytest.mark.parametrize("vals", [("good", True), ("bad", False)])
+def test_instance(cls, vals):
+    # test instantiation of abstract class
```

Comment:
So, I believe I've gotten the structure correct, but now I'm getting `redefining name 'create_config' from outer scope` error and its not creating the temp config values correctly. I've pushed those changes for following, although its not complete and differences not yet resolved

Reply:
`redefining ... from outer scope` is expected and should be disabled at the top of the file. Browse around the other test modules and you'll see this warning being disabled in many of them. `pylint` doesn't like to see a module-level name (the `create_config()` function in this case) shadowed/hidden by an argument to a function. But it's fine for us in this case.

I'll pull your code and have a look at the tests.

Reply:
In case it's helpful to anyone, I think I can motivate `pylint`'s concern a little more. Consider
```
>>> def foo() -> str:
...     return "foo"
... 
>>> def bar(foo) -> str:
...     return f"You gave me {foo}"
... 
>>> 
```
`pylint` would complain about the redefinition of `foo`. In the module scope `foo` is a function. But inside `bar()`, `foo` is an argument, whatever the caller passes in. The concern is that `bar()` would no longer be able to call `foo()` because, in its scope, `foo` has been redefined; `foo()` is now inaccessible to `bar()`.